text/tests/unittests/automm/configs/model/hf_text.yaml

```diff
@@ -0,0 +1,20 @@
+model:
+  names: "hf_text"
+  hf_text:
+#    checkpoint_name: "google/electra-base-discriminator"
+    checkpoint_name: 'microsoft/deberta-v3-small'
+#     checkpoint_name: 'bert-base-uncased'
+#    checkpoint_name: 'xlm-roberta-base'
+#    checkpoint_name: 'microsoft/deberta-base'
+#    checkpoint_name: 'roberta-base'
+#    checkpoint_name: "distilbert-base-uncased"
+#    checkpoint_name: "bert-base-chinese"
+#    checkpoint_name: "gpt2"
```

Comment:
remove comments + add newline.

Reply:
Are you seeing an old commit? These were already fixed.

Reply:
Thanks. Fixed this time.